TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -3876,6 +3876,18 @@ Access control is discussed in detail in <doc:AccessControl>.
   Declarations marked with the `public` access-level modifier can also be accessed (but not subclassed)
   by code in a module that imports the module that contains that declaration.
 
+- term `package`:
+  Apply this modifier to a declaration
+  to indicate that the declaration can be accessed
+  only by code in the same package as the declaration.
```

Comment:
Change passive voice to active:
```suggestion
  to indicate that only code in the same package 
  as the declaration can access it.
```

Reply:
Suggest stet.  This wording is parallel to the existing wording for the other access levels.  The focus of this bulleted list is on the declaration, not on what's using it, so these descriptions tend to be easier to read when their wording follows that ordering.

That said, this usage doesn't quite rise to the level of "undefined, unimportant actor" that the TSPL [style guide](https://github.com/apple/swift-book/blob/main/Style.md#passive-voice) notes is a place where the book uses passive voice — there is an actor here, although it's somewhat vaguely defined "other code somewhere else".  So there's room for argument that we should rewrite the entire list.

Reply:
Oh, thanks for sharing the TSPL style guide! I've bookmarked that for future reference.

The level of effort to update the entire list to be active doesn't seem worth it. I'm ok with leaving as passive to be parallel with the other items.